src/zope/configuration/tests/test_docs.py

```diff
@@ -77,21 +45,6 @@ def test_docs(self):
         'zopeconfigure',
     )
 
-    paths = [os.path.join(docs, f) for f in doc_files_to_test]
-    paths += [os.path.join(api_docs, f) for f in api_files_to_test]
-
-    m = manuel.ignore.Manuel()
```

Comment:
Can you clean up the `manuel` import from setup.py as well?

Reply:
Fixed in https://github.com/zopefoundation/zope.configuration/pull/61/commits/4bd80e2ef02556f1dc29641d4b3938fa37d01d07